test/optimus/scientific/ScientificTests.wdl

```diff
@@ -1,26 +1,70 @@
+task TestBam {
+
+  # Compare the tagged/aligned bam generated by Optimus to expected values derived from running Cell Ranger on the same data"
 
-# note that this is not a great test because it localizes the .bam file
-# however it is a good placeholder while we put together the scientific tests
-# themselves, as these files will be relatively small in the testing data.
-task TestBamRecordNumber {
   File bam
-  Int expected_records
+
+  Int expected_records = 3055752
```

Comment:
Could you pass in these expected values from the cell ranger wdl running on the same data instead of hard-coding them here?

Reply:
I am not allowed to check in the cell ranger wdl yet, but eventually the plan is to calculate these on the fly, yep. 👍